ci-operator/step-registry/baremetalds/e2e/test/baremetalds-e2e-test-commands.sh

```diff
@@ -39,6 +39,25 @@ function set_test_provider() {
     fi
 }
 
+function mirror_release_image_for_disconnected_upgrade() {
+    # All IPv6 clusters are disconnected and
+    # release image should be mirrored for upgrades.
+    if [[ "${DS_IP_STACK}" == "v6" ]]; then
+      # shellcheck disable=SC2087
+      ssh "${SSHOPTS[@]}" "root@${IP}" bash - << EOF
+MIRRORED_RELEASE_IMAGE=${DS_REGISTRY}/localimages/local-release-image
+DIGEST=\$(oc adm release info --registry-config ${DS_WORKING_DIR}/pull_secret.json ${OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE} --output=jsonpath="{.digest}")
+oc adm release mirror --registry-config ${DS_WORKING_DIR}/pull_secret.json --from=${OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE} --to=\${MIRRORED_RELEASE_IMAGE} --apply-release-image-signature
+echo "OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE=\${MIRRORED_RELEASE_IMAGE}@\${DIGEST}" >> /tmp/disconnected_mirror.conf
+EOF
+
+      # shellcheck source=/dev/null
+      source <(ssh "${SSHOPTS[@]}" "root@${IP}" "cat /tmp/disconnected_mirror.conf")
```

Comment:
I think that adding an echo here showing the new value of `OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE` could be helpful for troubleshooting/debugging

Reply:
I thought same thing but there seems not need echo because `OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE` can be seen `to-image` in `openshift-tests run upgrade-all` command.

Reply:
But not as `OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE`. Maybe we can add, what do you think?

Reply:
I think that moving ` mirror_release_image_for_disconnected_upgrade` after the `set -x` (to make it less verbose) and adding a message saying that the release image have been mirrored due the disconnected environment will be enough

Reply:
@andfasano I changed that way, could you please have a look at once again?, thanks.